mne/epochs.py

```diff
@@ -2965,6 +2812,9 @@ def __init__(self, data, info, events=None, tmin=0, event_id=None,
             self._detrend_offset_decim(e, detrend_picks)
         self.drop_bad()
 
+        # also create an empty Annotations object
+        self._annotations = None
```

Comment:
This should be done by the `BaseEpochs` class I think

Reply:
Good point. Moved all these into BaseEpochs.